opentelekomcloud/resource_opentelekomcloud_compute_bms_server_v2.go

```diff
@@ -364,7 +364,7 @@ func resourceComputeBMSInstanceV2Create(d *schema.ResourceData, meta interface{}
 	d.SetId(server.ID)
 
 	// Set bms sepcific tag
-	bmsClient, err := config.bmsClient(GetRegion(d, config))
+	bmsClient, err := config.computeV2Client(GetRegion(d, config))
```

Comment:
computeV2Client covers bms?

Reply:
`config.bmsClient` is literally the same:
```go
// bmsClient used to access the v2.1 bms Services i.e. servers, tags.
func (c *Config) bmsClient(region string) (*golangsdk.ServiceClient, error) {
	return openstack.NewComputeV2(c.HwClient, golangsdk.EndpointOpts{
		Region:       region,
		Availability: c.getHwEndpointType(),
	})
}
```